addons/mail/static/src/views/web/fields/many2many_tags_email/many2many_tags_email.js

```diff
@@ -121,8 +122,10 @@ export class FieldMany2ManyTagsEmail extends Many2ManyTagsField {
     }
 
     async updateRecipient(newEmail, partnerId) {
-        await this.orm.write("res.partner", [partnerId], { email: newEmail });
-        return this.props.record.data[this.props.name].addAndRemove({ reload: true });
+        const list = this.props.record.data[this.props.name];
+        const partnerRecord = list.records.find((r) => r.resId === partnerId);
+        partnerRecord.canSaveOnUpdate = true;
```

Comment:
Why do you need to specify this here? From what I understand it is set to `true` if there is no parentRecord for a specific one. But I don't see how a `res.partner` could have a parentRecord, as it doesn't really make sense to me. 
Is it more of a safeguard or is there a need?

Reply:
Hello @thjo-odoo 

Thanks for the review.

The edited partner is a record in the many2many. In this context, `parentRecord` is the "main" record (the one with the many2many field). This condition is there to prevent from writing directly on a subrecord (typically in form views), where you instead want the main record to be saved. It's not related to the "parent" relationship of the res.partner.

Reply:
Ok that makes sense to me. 
As unfortunately I am no R+ hero and ABD is off this week, I don't know how you want to handle the R+.
Do you want someone from your team to also check the PR, or do I summon someone from mine?